conf/mendel.config

```diff
@@ -2,21 +2,23 @@
 params {
   config_profile_name = 'MENDEL'
   config_profile_description = 'GMI MENDEL cluster profile provided by nf-core/configs'
-  config_profile_contact = 'Philipp H (@phue)'
+  config_profile_contact = 'Patrick Hüther (@phue)'
   config_profile_url = 'http://www.gmi.oeaw.ac.at/'
 }
 
-singularity {
-  enabled = true
+manifest {
+  nextflowVersion = '>=19.01.0'
 }
 
 process {
-  beforeScript = 'module load Singularity'
-  executor = 'pbs'
+  beforeScript = {'module load Singularity; module load Miniconda3'}
```

Comment:
Is there a reason why you a.) load both and b.) not an explicit version of the tool?

Reply:
a) loading both leaves more options, our Singularity deployment has issues and most users will use conda. However, some people (including me) like containers.
b) In our environment, the latest version is loaded by default. I know this is not best practice but these things rarely get updated here and never caused breakage so far